src/main/java/com/assure/movie/service/MemberService.java

```diff
@@ -32,6 +34,9 @@ public Member getMember(Long id) throws NotFoundErrorException {
                 new NotFoundErrorException("Not found member with id: "+ id)
         );
     }
+    public List<Member> getMembers() {
+        return this.memberRepository.findAll();
+    }
     @Transactional
     public Member saveMember(Member member) {
         return this.memberRepository.save(member);
```

Comment:
What will happen if the _member_ object is null?

Reply:
if the _member_ object is null an exception will be throw from the JPA Repository